### PR TITLE
node domain

### PR DESCRIPTION
**Pull request by Jodge**
# Description

medic/medic#5300

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by Jodge** [comment]
Accidentally closed

**Comment by Jodge** [comment]
@garethbowen there seems to be no clear way of testing streams, any idea around that?

**Comment by Jodge** [comment]
@garethbowen please review

File: api/src/controllers/export-data.js [hunk 1]
```diff
@@ -92,25 +91,28 @@ module.exports = {
       })
       .then(() => auth.check(req, getExportPermission(req.params.type)))
       .then(() => {
-        writeExportHeaders(res, req.params.type, formats.csv);
+        return new Promise(resolve => {
```
Comment (garethbowen) [inline api/src/controllers/export-data.js]: I don't think this promise does anything - if I remove it everything works as expected. Can it be removed?

**Pull request merged**